apiserver/common/secrets/watcher.go

```diff
@@ -20,19 +21,23 @@ type secretBackendModelConfigWatcher struct {
 	src           watcher.NotifyWatcher
 	backendGetter SecretBackendGetter
 	logger        logger.Logger
+	modelUUID     coremodel.UUID
 
 	currentSecretBackend string
 }
 
-func newSecretBackendModelConfigWatcher(ctx context.Context, backendGetter SecretBackendGetter, src watcher.NotifyWatcher, logger logger.Logger) (watcher.NotifyWatcher, error) {
+func newSecretBackendModelConfigWatcher(
```

Comment:
This needs to be renamed now. Same with `processModelChange`

In fact, if we are watching the model_secret_backend table directly, we don't even need this wrapper.

Reply:
You are right, because the table we are watching only has two fields.